### DISCUSSION TITLE
Logging multiple variables from different sensors

### DISCUSSION THREAD

**Discussion by SkywalkerJLi** [bitcraze/discussions#845] [Q&A]
I was wondering if it was possible to log both accelerometer sensor measurements and Kalman state estimation data at the same time. I know how to get the data individually using 

lg_kal = LogConfig(name='kalman', period_in_ms = 1000)
lg_kal.add_variable('kalman.stateX')
lg_kal.add_variable('kalman.stateY')
lg_kal.add_variable('kalman.stateZ')` 

and 

with SyncLogger(scf, lg_kal) as logger:
           for log_entry in logger:

but don't know how to log acc data as I'm logging Kalman data. Has anyone done something similar?

**Comment by krichardsson** [comment]
Hi,
you can mix and match any log variables in the same LogConfig
```python 
lg_kal = LogConfig(name='kalman', period_in_ms = 1000)
lg_kal.add_variable('kalman.stateX')
lg_kal.add_variable('kalman.stateY')
lg_kal.add_variable('kalman.stateZ')
lg_kal.add_variable('acc.x')
lg_kal.add_variable('acc.y')
lg_kal.add_variable('acc.z')
```

There is a limit to how much data that can be logged using one config (due to packet sizes), but you can use more log configs if you want to in the SyncLogger by putting them in a list.

```python
with SyncLogger(scf, [lg_kal, my_other_log_config]) as logger:
    for log_entry in logger:
        # ...
```
